timelock-server/src/main/java/com/palantir/atlasdb/timelock/TimeLockServerLauncher.java

```diff
@@ -70,21 +67,16 @@ public static void main(String[] args) throws Exception {
 
     @Override
     public void initialize(Bootstrap<CombinedTimeLockServerConfiguration> bootstrap) {
-        MetricRegistry metricRegistry = SharedMetricRegistries.getOrCreate(
-                "AtlasDbTest" + UUID.randomUUID().toString());
+        MetricRegistry metricRegistry = SharedMetricRegistries.getOrCreate("AtlasDbTest" + UUID.randomUUID());
         bootstrap.setMetricRegistry(metricRegistry);
-        bootstrap.setObjectMapper(ObjectMappers.newServerObjectMapper());
-
-        bootstrap.getObjectMapper().registerSubtypes(NonBlockingFileAppenderFactory.class);
-        bootstrap.getObjectMapper().setSubtypeResolver(new DiscoverableSubtypeResolver());
-        bootstrap.getObjectMapper().registerModule(new Jdk8Module());
+        bootstrap.setObjectMapper(
+                ObjectMappers.newServerObjectMapper().setSubtypeResolver(new DiscoverableSubtypeResolver()));
```

Comment:
I don't understand why we are getting rid of Jdk8Module and JavaTimeModule here. Other than that everything looks good!

Reply:
Those are added by `ObjectMappers.newServerObjectMapper()`.